addons/api/CfgFunctions.hpp

```diff
@@ -29,7 +29,6 @@ class CfgFunctions {
 
             PATHTO_FNC(filterUnitLoadout);
 
-            PATHTO_FNC(setItemRadioReplacement);
```

Comment:
Should use https://github.com/IDI-Systems/acre2/blob/1f363d002b337c8451f6a09ec6fc5b355aca9e5c/addons/main/script_macros.hpp#L151.